examples/nodejs-express-launchdarkly/lib/launchdarkly.js

```diff
@@ -0,0 +1,37 @@
+import * as ld from "@launchdarkly/node-server-sdk";
+
+// Initialize LaunchDarkly client
+const client = ld.init(process.env.LAUNCHDARKLY_SDK_KEY);
+
+export const getArcjetConfig = async () => {
+  // Wait for the LaunchDarkly client to be initialized
+  await client.waitForInitialization({ timeout: 500 });
+
+  // Set the user context for LaunchDarkly - in this example, every user is treated the same.
+  const context = { key: "guest" };
+
+  // Get the latest configuration from LaunchDarkly
+  const shieldMode = await client.variation("shieldMode", context, "LIVE");
```

Comment:
Is it worth mentioning that the 3rd argument is the default? Maybe you could make a "defaults" object at the top and pull from it instead of inlining them